docs/docs-content/release-notes.md

```diff
@@ -11,11 +11,46 @@ sidebar_custom_props:
 tags: ["release-notes"]
 ---
 
+## October 10, 2023 - Release 4.0.19
+
+This release contains the following enhancements and minor bug fixes.
+
+### Enhancements
+
+- You can now modify registry mapping patterns that were previously hardcoded in the Palette CLI and Helm Charts.
+
+- You can now disable local accounts on existing Azure Kubernetes Services (AKS) clusters at the Kubernetes layer by setting the `managedControlPlane.aadProfile.disableLocalaccounts` parameter to `true`. Using the same parameter, you can also create a new cluster with local accounts disabled and re-enable local accounts on existing clusters.
+
+- You can now configure OpenID Connect (OIDC) for virtual clusters. Refer to [Configure OIDC for a Virtual Cluster](clusters/palette-virtual-clusters/configure-oidc-virtual-cluster.md). 
+
+- Cluster administrators can now deploy and manage virtual clusters using add-on profiles.
```

Comment:
In cluster mode, users will be able to deploy addon profiles on the virtual clusters.

Reply:
Added: In cluster mode, you can now deploy add-on profiles on virtual clusters.